src/exercises/day-1/for-loops-bonus.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+fn transpose(matrix: &[&[i32]; 3]) -> [[i32; 3]; 3] {
+    let len_r = matrix[0].len();
+    let len_c = matrix.len();
+    let mut t_matrix = [[0;3];3];
```

Comment:
Please name this `result` like in the other solution — it might not be a super name, but at least it's consistent.